src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6500,6 +6500,41 @@ IOperation visitInvalidOperationExpression(IInvalidOperation invalidOperation)
             }
         }
 
+        public override IOperation VisitReDim(IReDimOperation operation, int? argument)
```

Comment:
>VisitReDim [](start = 35, length = 10)

Please use StartVisitingStatement/FinishVisitingStatement helpers as we do when other statements are rewritten.